### PR TITLE
Translations (Valyx/Phares)

### PR DESCRIPTION
**Pull request by Mi12yuRe**
Added translation to Va / Ph's Normal Attack Combo (Test Passed)
& Skill Identifiers (S1&2)

FYI - Skill Identify test **FAILED** 
Pls help to check if there's any missing trans. THX

### CODE REVIEW

**Comment by Mi12yuRe** [comment]
Deleted surplus skill Identifiers for mode 144 (Valyx)
Skill Identifier for mode 16 was deleted by mistake - Added back.

File: .data/custom/WebsiteText@en.json [hunk 1]
```diff
@@ -179,6 +179,18 @@
...
+    "id": "SKILL_IDENTIFIER_s1_mode_144",
+    "text": "S1"
+  },
+  {
+    "id": "SKILL_IDENTIFIER_s1_mode_145",
+    "text": "S1 (Normal)"
+  },
+  {
+    "id": "SKILL_IDENTIFIER_s1_mode_146",
+    "text": "S1 (Enhance)"
```
Comment (RaenonX) [inline .data/custom/WebsiteText@en.json:192]: Dragon Drive - not "Enhance"

File: .data/custom/WebsiteText@en.json [hunk 2]
```diff
@@ -179,6 +179,18 @@
     "id": "SKILL_IDENTIFIER_s1_mode_141",
     "text": "S1 (Enhance)"
```
Comment (RaenonX) [inline .data/custom/WebsiteText@en.json:180]: Dragon Drive - not "Enhance"

File: .data/custom/WebsiteText@en.json [hunk 3]
```diff
@@ -387,6 +399,14 @@
     "id": "SKILL_IDENTIFIER_s2_mode_141",
     "text": "S2 (Enhance)"
```
Comment (RaenonX) [inline .data/custom/WebsiteText@en.json:400]: Dragon Drive - not "Enhance"

File: .data/custom/WebsiteText@en.json [hunk 4]
```diff
@@ -387,6 +399,14 @@
...
   },
+  {
+    "id": "SKILL_IDENTIFIER_s2_mode_145",
+    "text": "S2 (Normal)"
+  },
+  {
+    "id": "SKILL_IDENTIFIER_s2_mode_146",
+    "text": "S2 (Enhance)"
```
Comment (RaenonX) [inline .data/custom/WebsiteText@en.json:408]: Dragon Drive - not "Enhance"

**Review by RaenonX — CHANGES_REQUESTED** [review]
Check all `(Enhance)` usages and replace it accordingly

**Review by RaenonX — APPROVED** [review]

**Pull request merged**